src/mainboard/clevo/adl-p/ramstage.c

```diff
@@ -19,8 +29,7 @@ void mainboard_silicon_init_params(FSP_S_CONFIG *params)
 	params->PchSerialIoI2cSdaPinMux[1] = 0x1947c606; // GPP_H6
 	params->PchSerialIoI2cSclPinMux[1] = 0x1947a607; // GPP_H7
 
-	params->SataPortDevSlpPinMux[0] = 0x59673e0c; // GPP_H12
-	params->SataPortDevSlpPinMux[1] = 0x5967400d; // GPP_H13
+	params->SataPortDevSlpPinMux[1] = 0x59673e0c; // GPP_H12
```

Comment:
We have to revert this change as it will not work anyways. The DEVSLP pins must match SATA ports 1 to 1. Would also add a comment that design is broken.

Reply:
Added